cfme/tests/infrastructure/test_set_default_filter.py

```diff
@@ -25,7 +25,7 @@ def unset_default_host_filter():
     appliance.server.logout()
     appliance.server.login_admin()
     navigate_to(host_collection, 'All')
-    assert view.filters.navigation.currently_selected[0] == 'Status / Running (Default)'
+    assert view.filters.navigation.currently_selected[-2] == 'Status / Running (Default)'
```

Comment:
REQUIRED : If I got you right, currently_selected returns list of all filters. This is wrong. So, currently_selected should be fixed along with test.

Reply:
You are right, currently_selected should return single selected value not list. Changing it! :) 👍